### PR TITLE
Update contexts.md

### PR DESCRIPTION
**Pull request by thomasr8**
Added additional content to note about additional requirements if using a BitBucket organization.

# Description
A brief description of the changes.

# Reasons
A link to a GitHub and/or JIRA issue (if applicable).
Otherwise, a brief sentence about why you made these changes.

### CODE REVIEW

File: jekyll/_cci2/contexts.md [hunk 1]
```diff
@@ -41,11 +41,13 @@ If you find you need to rename an org or repo that you have previously hooked up
 
 ## Creating and Using a Context
 
-1. Using the new version of the CircleCI application, navigate to the Organization Settings page by clicking on the link in the sidebar. **Note:** Any organization member can create a context, but only organization administrators can restrict it with a security group.
+1. Using the new version of the CircleCI application, navigate to the Organization Settings page by clicking on the link in the sidebar. 
+
+**Note:** Organization members can create a context, but only organization administrators can restrict it with a security group. The one exception to this case is BitBucket organizations, which require a user to have the `create repositories` workkspace permission, regardless of their other permissions on the workspace or the repositories it contains.
```
Comment (rosieyohannan) [inline jekyll/_cci2/contexts.md:46]:
```suggestion
    **Note:** Organization members can create a context, but only organization administrators can restrict it with a security group. The one exception to this case is BitBucket organizations, which require a user to have the `create repositories` workkspace permission, regardless of their other permissions on the workspace or the repositories it contains.
```

File: jekyll/_cci2/contexts.md [hunk 2]
```diff
@@ -41,11 +41,13 @@ If you find you need to rename an org or repo that you have previously hooked up
...
     
     ![Contexts]({{ site.baseurl }}/assets/img/docs/org-settings-contexts-v2.png)
 
-    **Note: If you are using the old version of the CircleCI application**, you can access the Organization Settings using the following URL format: `https://circleci.com/gh/organizations/<org>/settings#contexts`, substituting your organization/GitHub username. If you are using CircleCI Server, Organization Settings can still be accessed as normal from the main navigation.
+**Note: If you are using the old version of the CircleCI application**, you can access the Organization Settings using the following URL format: `https://circleci.com/gh/organizations/<org>/settings#contexts`, substituting your organization/GitHub username. If you are using CircleCI Server, Organization Settings can still be accessed as normal from the main navigation.
```
Comment (rosieyohannan) [inline jekyll/_cci2/contexts.md:50]: This was indented to keep the numbered steps flowing - left-aligning a paragraph will force numbering to start again at 1

File: jekyll/_cci2/contexts.md (hunk 1 above)
Comment (rosieyohannan) [reply jekyll/_cci2/contexts.md:46]: Not sure this fixes it, would have to check locally, but the para needs indenting to keep the numbered steps flowing correctly.

**Comment by rosieyohannan** [comment]
Content looks great, just formatting needs looking at :-)

File: jekyll/_cci2/contexts.md (hunk 1 above)
Comment (yaningo) [reply jekyll/_cci2/contexts.md:46]: There is a small typo:
**_workkspace_** instead of **_workspace_**, just after `create repositories`

Comment (thomasr8) [reply jekyll/_cci2/contexts.md]: fixed both issues 👍

File: jekyll/_cci2/contexts.md (hunk 2 above)
Comment (thomasr8) [reply jekyll/_cci2/contexts.md]: Fixed

**Pull request merged**